src/main/webapp/WEB-INF/tags/instructor/results/questionPanel.tag

```diff
@@ -0,0 +1,84 @@
+<%@ tag description="instructorFeedbackResults - by question" %>
+<%@ taglib uri="http://java.sun.com/jsp/jstl/core" prefix="c" %>
+
+<%@ tag import="teammates.common.util.Const" %>
+
+<%@ taglib tagdir="/WEB-INF/tags/instructor/results" prefix="results" %>
+
+<%@ attribute name="showAll" type="java.lang.Boolean" required="true" %>
+<%@ attribute name="questionIndex" type="java.lang.Integer"%>
+<%@ attribute name="shouldCollapsed" type="java.lang.Boolean" required="true" %>
```

Comment:
if I keep it as an Integer, then jstl throws an exception if I pass a string into it, so it's better to keep it as an integer for safety.